documentation/src/main/asciidoc/topics/ref_server_realm_property.adoc

```diff
@@ -8,7 +8,11 @@ Passwords can be pre-digested with the `DIGEST-MD5` and `DIGEST` authentication
 
 [NOTE]
 ====
-Properties files contain headers that associate them with security realms in {brandname} Server configuration.
+You can avoid authentication issues that relate to a property file by using the {brandname} CLI to enter the correct security realm name to the file. 
+
```

Comment:
Please remove the empty lines. In general place each sentence on its own line but don't leave any empty lines between.

Reply:
This will go to downstream docs once it's merged (the attributes will be substituted from downstream file).
Regarding starting each sentence on its won line - this applies to all our docs. You can check few adoc files and see the format